src/pages/blog.js

```diff
@@ -14,9 +14,9 @@ export default function Blog({ data }) {
           .filter(post => post.node.frontmatter.title.length > 0)
           .map(({ node: post }) => {
             return (
-              <div className="blog-post-preview">
+              <div className={style.blogPostPreview} onClick={() => navigate(post.frontmatter.path)}>
```

Comment:
here I have set up the blog list elements to have a full div onClick to navigate and converted to using the navigate function instead of the Link component to make things a bit more inline with current web standards